src/views/HomeView.vue

```diff
@@ -42,18 +72,114 @@
 </template>
 
 <script>
-import { mapGetters } from 'vuex';
+import _ from "lodash";
+import { mapGetters } from "vuex";
+
 export default {
   computed: {
-    ...mapGetters(['appData'])
+    ...mapGetters({
+      appData: "appData",
+      latestPosts: "blog/getLatestPosts"
+    }),
+    carouselSlides() {
+      return this.latestPosts.slice(0, 3);
+    },
+    otherLatestPosts() {
+      return this.latestPosts.slice(3, 6);
+    },
+    currentSlidePost() {
+      return this.carouselSlides[this.currentSlide];
+    }
   },
-  mounted() {
-    this.$store.dispatch('blog/getBlogPosts');
+  data() {
+    return {
+      currentSlide: 0
```

Comment:
initial slide is always 0. Data attribnutes can be manipulated after initial load so this is not always 0 after mounting the page